packages/google_sign_in/google_sign_in_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.1.3
```

Comment:
You'll need to move all of the `google_sign_in_platform_interface` (or all of the `google_sign_in`) changes to another PR; this is tripping our check against changing code in multiple parts of a federated plugin at the same time. (In this case it happens to be safe, but it's a sufficiently dangerous thing in general that our CI actually enforces not doing it in most cases.)

Just leave a link to the new PR here and I can make sure that the new PR gets reviewed along with this one.

Reply:
I created a new PR one: https://github.com/flutter/plugins/pull/5669